### DISCUSSION TITLE
EdgeXFoundary - in beaglebone black

### DISCUSSION THREAD

**Discussion by yogeshgopal304** [edgexfoundry/community#170] [Q&A]
Hey everyone,
I'm looking to install EdgeX Foundry on a BeagleBone Black. Any pointers on how to do this? Also, is there anything related to this on GitHub?

**Comment by cloudxxx8** [comment]
The default deployment approach of EdgeX is dock-compose or snap. You just need to install docker docker-compose or snap on your OS first.
https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedDockerUsers/
https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSnapUsers/

If your environemtn doesn't support docker or snap, you need to build the binary via Makefile on your own.
By the way, the community images doesn't support arm32.  If your environment is arm32, you need to build your own images and verify it.
